src/sql/workbench/contrib/dashboard/browser/widgets/explorer/explorerWidget.component.ts

```diff
@@ -122,6 +148,7 @@ export class ExplorerWidget extends DashboardWidget implements IDashboardWidget,
 					});
 					this._treeDataSource.data = profileData;
 					this._tree.setInput(new ExplorerModel());
+					this.setLoadingStatus(false);
 				},
 				error => {
 					(<HTMLElement>this._el.nativeElement).innerText = nls.localize('dashboard.explorer.databaseError', "Unable to load databases");
```

Comment:
Should set loading status to false here too (but probably have it announce that an error occurred instead of loading completed)